### PR TITLE
feat(paginate): expose paginate method

### PR DESCRIPTION
**Pull request by joffreyBerrier**
Expose `paginateToDay` to allow you to paginate on the day you want

### CODE REVIEW

File: example/src/App.vue [hunk 1]
```diff
@@ -199,6 +199,11 @@ const pushBookingDates = () => {
     }
   );
 };
+onMounted(() => {
+  console.log(calendar.value);
```
Comment (joffreyBerrier) [inline example/src/App.vue:203]: It doesn't work :/

Comment (harlan-zw) [reply example/src/App.vue:203]: So calendar.value is null here?

Comment (joffreyBerrier) [reply example/src/App.vue:203]: yep :/

File: src/components/Calendar.vue [hunk 2]
```diff
@@ -1006,6 +994,9 @@ const getBooking = (day: Day): FlatBooking | null => {
 const getBookingType = (day: Day): string | null => {
   return getBooking(day)?.key || null;
 };
+
+// Define expose
+defineExpose({ activeIndex, paginateToDay });
```
Comment (joffreyBerrier) [inline src/components/Calendar.vue:999]: @harlan-zw I expose the paginateToDay method, and activeIndex just for testing

**Pull request closed without merging**